src/raised-button.jsx

```diff
@@ -238,6 +243,9 @@ const RaisedButton = React.createClass({
       labelPosition: 'after',
       fullWidth: false,
       primary: false,
+      rippleStyle: {
```

Comment:
What about removing the default value and using `rippleStyle` as any other `style` property (that will override the default style)?